docs/developer-setup/github-authentication.md

```diff
@@ -0,0 +1,55 @@
+# Authenticating with the Github Packages Registry
+
+Some of the dependencies declared for this project include packages stored on the
+[Github Organisation's Packages Registry]. The [Github workflow files] pass secrets through
+github action input variables which in turn set the relevant `GITHUB_ACTOR` and `GITHUB_TOKEN`
+environment variables referred to within the [Project's Gradle settings]. This approach is less than
+ideal for local development so with this in mind, what can a developer do in order to complete
+their set up?
+
+## The what: property keys
+
+There are two gradle properties used to authenticate with the Github packages registry for local
+development:
+
+- `gpr.user`: The developer's Github account name. Associated with the `GITHUB_ACTOR` environment
+  variable.
+- `gpr.token`: The developer's Personal Access Token (PAT), generated in the
+  [Github developer settings]. Associated with the `GITHUB_TOKEN` environment variable. This token
+  must only have the `packages:read` permission, as the github workflow files handle uploading
+  artefacts.
+
+The proceeding command outputs the existing Personal Access Token, if the
+[Github Command-Line Interface (CLI) tool] is installed:
+
+```shell
+gh auth token
+```
+
+## The where: `gradle.properties`
+
+Because we no longer store credentials within the same directory structure as the git repository
```

Comment:
Small thing but best not to refer back to an old situation as that will quickly get out of date as new developers never will store credentials within the same directory. 
A more future-proof wording might be:

It is bad practice to store credentials within the same directory structure as the git repository. Instead these gradle properties should be stored within Gradle's user configuration, found on your local laptop
at `~/.gradle/gradle.properties`.

Reply:
That's understandable. However, I take issue with some of the wording you've suggested due to it not conforming with the google developer documentation style guide, such as the use of passive voice (should be stored).

I'm sure that when developers come to implement this within the various repositories, they'll find alternative ways of framing it in a way that meets everyone's expectations